kmip/core/enums.py

```diff
@@ -330,10 +330,9 @@ class SecretDataType(Enum):
     PASSWORD = 0x00000001
     SEED     = 0x00000002
 
-
 # 9.1.3.2.10
 class OpaqueDataType(Enum):
-    pass
+    NONE = 0x00000000 # Not defined by the standard, but we need something
```

Comment:
Good point, the spec says values not defined are reserved (9.1.3.2), so lets go with  0x80000000 here